sdks/wasm/Mono.WebAssembly.DebuggerProxy/MonoProxy.cs

```diff
@@ -561,38 +591,56 @@ async Task<Result> EnableBreakPoint (SessionId sessionId, Breakpoint bp, Cancell
 
 			if (ret_code.HasValue) {
 				bp.RemoteId = ret_code.Value;
-				bp.State = BreakPointState.Active;
+				bp.State = BreakpointState.Active;
 				//Log ("verbose", $"BP local id {bp.LocalId} enabled with remote id {bp.RemoteId}");
 			}
 
 			return res;
 		}
 
-		async Task LoadStore (SessionId sessionId, CancellationToken token)
+		async Task<DebugStore> LoadStore (SessionId sessionId, CancellationToken token)
```

Comment:
This should be moved into `ExecutionContext`. And `internal DebugStore store;` in `ExecutionContext` be made private.

Reply:
It is currently split for work not in this commit but I agree